packages/node-core/src/indexer/storeCache/cacheMmr.spec.ts

```diff
@@ -46,7 +46,8 @@ describe('CacheMMR', () => {
     const spy = jest.spyOn(db, 'setLeafLength');
 
     await cacheDb.setLeafLength(100);
-    expect(spy).toBeCalledTimes(1);
+    //there will be no call to db.setLeafLength
```

Comment:
` await cacheDb.flush({afterCommit:jest.fn()} as any,undefined)` 

We can use this, so db will setLeafLength once